.pa11yci

```diff
@@ -0,0 +1,10 @@
+{
+    "defaults": {
+        "ignore": [
+            "WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail"
+        ]
+    },
+    "urls": [
+        "http://localhost:8000/"
```

Comment:
We definitely don't need this, as pa11y will hit the homepage already since it's in the sitemap. I tested deleting the "urls" property altogether locally and confirmed it still hit `localhost:8000/`.